VM/DockerCerts-Generation/CreateDockerCerts.ps1

```diff
@@ -1,98 +1,101 @@
+param(
```

Comment:
I add these parameters to indicate the hostname and location to create the Uri of the certificates